Lib/test/test_long.py

```diff
@@ -1429,7 +1429,6 @@ def equivalent_python(byte_array, byteorder, signed=False):
             b'\x01\xff': -255,
             b'\x00\xff': -256,
             b'\xff\x00': 255,
-            b'\x00\x01': 256,
```

Comment:
This fixes a micro-bug flagged by ruff: this dictionary currently has two identical key/value pairs!

https://github.com/python/cpython/blob/162213f2db3835e1115178d38741544f4b4db416/Lib/test/test_long.py#L1421

https://github.com/python/cpython/blob/162213f2db3835e1115178d38741544f4b4db416/Lib/test/test_long.py#L1432